packages/playground/website/src/lib/tracking.ts

```diff
@@ -1,23 +1,94 @@
+import { StepDefinition } from '@wp-playground/blueprints';
+
 /**
  * Declare the global window.gtag function
  */
 declare global {
-    interface Window { gtag: any; }
+	interface Window {
+		gtag: any;
+	}
 }
 
 /**
  * Google Analytics event names
  */
-type GAEvent = 'load' | 'step';
+type GAEvent = 'load' | 'step' | 'install' | 'error';
 
 /**
  * Log a tracking event to Google Analytics
  * @param GAEvent The event name
  * @param Object Event data
  */
-export const logTrackingEvent = (event: GAEvent, data?: {[key: string]: string}) => {
-    if (typeof window === 'undefined' || !window.gtag) {
-        return;
-    }
-    window.gtag('event', event, data);
-}
+export const logTrackingEvent = (
+	event: GAEvent,
+	data?: { [key: string]: string }
+) => {
+	if (typeof window === 'undefined' || !window.gtag) {
+		return;
+	}
+	window.gtag('event', event, data);
+};
+
+/**
+ * Log Plugin install events
+ * @param step The Blueprint step
+ */
+export const logPluginInstallEvent = (step: StepDefinition) => {
+	const pluginData = (step as any).pluginData;
+	if (pluginData.slug) {
+		logTrackingEvent('install', {
+			plugin: pluginData.slug,
+		});
+	} else if (pluginData.url) {
+		logTrackingEvent('install', {
+			plugin: pluginData.url,
```

Comment:
Should we send URLs to GA?

I'm considering removing it because the data wouldn't be that useful for analytics.

Reply:
Would we want to see if a particular plugin URL was commonly used? Maybe it would be useful if we were tracking down particular failures? (I haven't looked at what GA gives us here, so these questions may not make sense)

Reply:
Analytics aren't meant for debugging, we have other ways of collecting errors. 
Also, the analytics data [is heavily sampled](https://support.google.com/analytics/answer/13331292?hl=en) so it's not a reliable source of information. 

I will remove URLs for now, we can always add them later.